safe/test/data/gisv4/exposure/places.geojson

```diff
@@ -2,17 +2,13 @@
 "features": [
-{ "type": "Feature", "properties": { "exposure_id": 1, "exposure_type": "hogwarts", "fake_field": 666, "absolute": 20, "ratio": 10 }, "geometry": { "type": "Point", "coordinates": [ 106.644120463110625, -6.147015351197105 ] } },
-{ "type": "Feature", "properties": { "exposure_id": 2, "exposure_type": "city", "fake_field": 666, "absolute": 0, "ratio": 0 }, "geometry": { "type": "Point", "coordinates": [ 106.643517753107972, -6.227308050721398 ] } },
-{ "type": "Feature", "properties": { "exposure_id": 3, "exposure_type": "village", "fake_field": 666, "absolute": 20, "ratio": 20 }, "geometry": { "type": "Point", "coordinates": [ 106.754416393596998, -6.230902960241568 ] } },
-{ "type": "Feature", "properties": { "exposure_id": 4, "exposure_type": "airport", "fake_field": 666, "absolute": 15, "ratio": 20 }, "geometry": { "type": "Point", "coordinates": [ 106.741458128539875, -6.197649109363691 ] } },
-{ "type": "Feature", "properties": { "exposure_id": 5, "exposure_type": "town", "fake_field": 666, "absolute": 1, "ratio": 2 }, "geometry": { "type": "Point", "coordinates": [ 106.76556652864619, -6.196750327497575 ] } },
-{ "type": "Feature", "properties": { "exposure_id": 6, "exposure_type": "room", "fake_field": 666, "absolute": 10, "ratio": 10 }, "geometry": { "type": "Point", "coordinates": [ 106.753210973591692, -6.172482639175887 ] } },
-{ "type": "Feature", "properties": { "exposure_id": 7, "exposure_type": "town", "fake_field": 666, "absolute": 5, "ratio": 5 }, "geometry": { "type": "Point", "coordinates": [ 106.754717748598338, -6.289316778482916 ] } },
-{ "type": "Feature", "properties": { "exposure_id": 8, "exposure_type": "city", "fake_field": 666, "absolute": null, "ratio": null }, "geometry": { "type": "Point", "coordinates": [ 106.753813683594359, -6.262956479734641 ] } },
-{ "type": "Feature", "properties": { "exposure_id": 9, "exposure_type": "secret_jail", "fake_field": 666, "absolute": 15, "ratio": 15 }, "geometry": { "type": "Point", "coordinates": [ 106.756827233607638, -6.119749021099483 ] } },
-{ "type": "Feature", "properties": { "exposure_id": 10, "exposure_type": "city", "fake_field": 666, "absolute": 5, "ratio": 2 }, "geometry": { "type": "Point", "coordinates": [ 106.815892813868103, -6.262956479734641 ] } },
-{ "type": "Feature", "properties": { "exposure_id": 11, "exposure_type": null, "fake_field": 666, "absolute": 0, "ratio": 0 }, "geometry": { "type": "Point", "coordinates": [ 106.801427773804321, -6.173081854759827 ] } },
-{ "type": "Feature", "properties": { "exposure_id": 11, "exposure_type": "unknown", "fake_field": 666, "absolute": 0, "ratio": 2 }, "geometry": { "type": "Point", "coordinates": [ 106.893801479188284, -6.118605054148777 ] } }
+{ "type": "Feature", "properties": { "Name": "Alubarna", "Type": "Capital City", "Population": 1000000, "Female": 270000, "Male": 730000 }, "geometry": { "type": "Point", "coordinates": [ 106.752992014058449, -6.171999008052889 ] } },
```

Comment:
You didn't like hogwarts? ;-)

Reply:
He is inserting One Piece reference :v

Reply:
:D

Reply:
I was thinking to use Asgard or Sakaar, but perhaps those are too obvious.